packages/xo-web/src/common/intl/messages.js

```diff
@@ -2380,6 +2380,8 @@ const messages = {
   proxyNetworkNetmaskPlaceHolder: 'Default to: {netmask}',
   proxySrPredicateInfo:
     'The select only contains SRs connected to at least one HVM-capable host',
+  proxyUpToDate: 'Up to date',
+  cannotGetProxyUpdates: 'Cannot get proxy updates',
```

Comment:
What does it mean for the user?

Reply:
It means that the updates cannot be fetched, i can add `please look your logs` if necessary.

Reply:
Does it mean that it failed to download the upgrades and therefore failed to upgrade the proxy?

Reply:
It's an error triggered on calling `getProxyApplianceUpdaterState`. We cannot know the cause because the error message is wrapped. We only know that the fetch of the proxy updates failed. The user can take a look to his logs to see what happens.

See https://github.com/vatesfr/xen-orchestra/pull/5167/files#diff-5c00d01ffc1fb5facb3a83a9a7688111R191 and https://github.com/vatesfr/xen-orchestra/pull/5167/files#diff-5c00d01ffc1fb5facb3a83a9a7688111R191

Reply:
`Cannot get updates state`?

Reply:
`Unable to check updates availability`?

Reply:
Done :+1: